src/main/java/pingis/controllers/TaskController.java

```diff
@@ -19,7 +19,7 @@
 import pingis.entities.ImplementationType;
 import pingis.entities.TaskImplementation;
 
-import pingis.entities.TmcSubmission;
+import pingis.entities.tmc.TmcSubmission;
```

Comment:
Maybe rename `TmcSubmission` to plain `Submission`, as `tmc.TmcSubmission` is kinda redundant.